tests/phpunit/elementor/modules/home/transformations/test-remove-sidebar-upgrade-for-pro-users.php

```diff
@@ -0,0 +1,65 @@
+<?php
+namespace Elementor\Tests\Phpunit\Elementor\Modules\Home\Transformations;
+
+use Elementor\Modules\Home\Transformations\Remove_Sidebar_Upgrade_For_Pro_Users;
+use ElementorEditorTesting\Elementor_Test_Base;
+
+class Test_Remove_Sidebar_Upgrade_For_Pro_Users extends Elementor_Test_Base {
```

Comment:
```suggestion
use PHPUnit\Framework\TestCase as PHPUnit_TestCase;

class Test_Remove_Sidebar_Upgrade_For_Pro_Users extends PHPUnit_TestCase {
```

Elementor_Test_Base loads a bunch of stuff you don't need. Please check if the plain PHPUnit_TestCase works ok?

Reply:
Updated. Including Tzvi's comment.